sql/item_mods.sql

```diff
@@ -37210,6 +37212,7 @@ INSERT INTO `item_mods` VALUES (27414,31,69);
 INSERT INTO `item_mods` VALUES (27414,68,72);
 INSERT INTO `item_mods` VALUES (27414,384,40);
 INSERT INTO `item_mods` VALUES (27414,386,55);
+INSERT INTO `item_mods` VALUES (27414,429,25);     -- Footwork+10% of 256 = 25.6
```

Comment:
Same as above, can anyone tell me if the divisor here should be the 256, or 1024? I am unsure where to look...